stylesheets/example-tooltip.scss

```diff
@@ -50,18 +50,27 @@ $shadow-color: rgba(31, 28, 28, 0.3);
     .btn {
       font-size: 13px;
       font-weight: 400;
-      color: $white;
-      background-color: #78A300;
+      color: #78A300;
+      background-color: transparent;
+      border: 1px solid #78A300;
       border-radius: 3px;
       height: 36px;
       padding: 0 20px;
+
+    }
+
+    .btn-next {
+      color: $white;
+      background-color: #78A300;
```

Comment:
Could you add some screenshots of what the tooltip looks like after adding the previous button and these style changes?